cmds/core/basename/basename_test.go

```diff
@@ -39,15 +41,19 @@ func TestRun(t *testing.T) {
 		},
 		{
 			flags: []string{},
-			out:   "Usage: basename NAME [SUFFIX]",
+			err:   errUsage,
 		},
 	}
 
 	// Table-driven testing
 	for _, tt := range tests {
 		t.Run(fmt.Sprintf("Using flags %s", tt.flags), func(t *testing.T) {
 			var out bytes.Buffer
-			run(&out, tt.flags)
+			err := run(&out, tt.flags)
+
+			if !errors.Is(err, tt.err) {
+				t.Fatalf("expected %v got %v", tt.err, err)
+			}
```

Comment:
```suggestion
			if err := run(&out, tt.flags); err != nil && !errors.Is(err, tt.err) {
			}
				t.Fatalf("expected %v got %v", tt.err, err)
			}
```

Reply:
the common form I've been recommended to use over the years is the got, want form
got %v, want %v
also, you can use errors.Is with nil, i.e., I believe the err != nil check is not needed. If you use errors.Is, you can also easily add tests where err is nil and should not be.

Reply:
I changed to got/want.